### ISSUE TITLE
Mekanism machines still broken

### ISSUE THREAD

**Issue by CheAle14** [CheAle14/mc-mod#31]

**Comment by CheAle14** [comment]
**Able to reproduce**
```
[22:03:40] [Server thread/WARN] [mlapi]: [WARNING] Playercount has dropped below 2, server will pause in 1 seconds
[22:03:41] [Server thread/WARN] [mlapi]: [WARNING] Playercount has dropped below 2, server will pause in 0 seconds
[22:03:41] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=60, y=67, z=613} = LOW, now: PULSE
[22:03:41] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=59, y=67, z=614} = DISABLED, now: PULSE
[22:03:41] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=60, y=68, z=614} = HIGH, now: PULSE
[22:03:41] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=61, y=67, z=614} = DISABLED, now: PULSE
[22:03:41] [Server thread/WARN] [mlapi]: Server has been paused, time is 1407540
[22:03:57] [Server thread/INFO] [net.minecraft.network.NetHandlerPlayServer]: Neutral lost connection: Disconnected
```
on relog, still paused:
```
[22:04:15] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer]: Neutral joined the game
[22:04:15] [Server thread/INFO] [FTBLib]: Event com.feed_the_beast.ftblib.events.SyncGamerulesEvent fired, cancelled: false
[22:04:15] [Server thread/INFO] [FTBLib]: Event com.feed_the_beast.ftblib.events.player.ForgePlayerLoggedInEvent fired, cancelled: false
[22:04:15] [Server thread/DEBUG] [mlapi]: Player Neutral has no team, is in global
[22:04:15] [Server thread/INFO] [mlapi]: Already paused, so not updating time we paused at, if you desire, it is: 1408223
[22:04:15] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=60, y=67, z=613} = PULSE, now: PULSE
[22:04:15] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=59, y=67, z=614} = PULSE, now: PULSE
[22:04:15] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=60, y=68, z=614} = PULSE, now: PULSE
[22:04:15] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=61, y=67, z=614} = PULSE, now: PULSE
[22:04:15] [Server thread/WARN] [mlapi]: Server has been paused, time is 1407540
```
hence, upon unpause:
```
[22:04:41] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=60, y=67, z=613} = PULSE
[22:04:41] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=59, y=67, z=614} = PULSE
[22:04:41] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=60, y=68, z=614} = PULSE
[22:04:41] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=61, y=67, z=614} = PULSE
[22:04:41] [Server thread/WARN] [mlapi]: Server has been unpaused, time reset, paused for 683
[22:04:41] [Server thread/INFO] [net.minecraft.server.dedicated.DedicatedServer]: Action succeeded
[22:04:42] [Server thread/WARN] [mlapi]: Unpausing all elements after some ticks.
[22:04:42] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=60, y=67, z=613} = PULSE
[22:04:42] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=59, y=67, z=614} = PULSE
[22:04:42] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=60, y=68, z=614} = PULSE
[22:04:42] [Server thread/INFO] [mlapi]: EnergizedSmelter @ BlockPos{x=61, y=67, z=614} = PULSE
[22:04:42] [Server thread/WARN] [mlapi]: Server has been unpaused, time reset, paused for 683
[22:04:53] [Server thread/DEBUG] [FML]: Gathering id map for writing to world save world
```

**Comment by CheAle14** [comment]
Thus, the issue *appears* to be that relogging, or indeed, having the tiles be loaded, then unloaded, then re-loaded, causes them to be added *twice* to the system, or perhaps overwritten - but either way, their control system is ultimately lost.

**Comment by CheAle14** [comment]
Saving thought:
When player joins, PauseHelper calls each FrozenWorld `Pause()` function, which *clears* all items in `Freezeables` - what it should instead do is go through each entity, and *add any non-existing ones*
Already implemented `IFrozenitem::Equals(IFrozenItem)`, so should be fairly trivial.

**Comment by CheAle14** [comment]
Closed in PR #37
